packages/core/src/Stores/client-store.js

```diff
@@ -792,6 +792,11 @@ export default class ClientStore extends BaseStore {
         return isMultipliersOnly(this.residence);
     }
 
+    @computed
+    get is_poi_dob_mismatch() {
+        return this.account_status?.status?.includes('poi_dob_mismatch') ?? false;
+    }
+
```

Comment:
Can you add this around line 500 close to the rest of the account statuses.

Reply:
Will refactor the code